docs/source/models.rst

```diff
@@ -5,18 +5,18 @@ MLflow Models
 
 An MLflow Model is a standard format for packaging machine learning models that can be used in a
 variety of downstream tools---for example, real-time serving through a REST API or batch inference
-on Apache Spark. The format defines a convention that lets you save a model in different "flavors" that can be
-understood by different downstream tools.
+on Apache Spark. The format defines a convention that lets you save a model in different "flavors" 
+that can be understood by different downstream tools.
 
 .. contents:: Table of Contents
   :local:
-  :depth: 2
+  :depth: 1
```

Comment:
I kind of liked the depth-2 here, it gives a good high-level view of what models we support, and concrete examples of what we're talking about:

![image](https://user-images.githubusercontent.com/1400247/45247984-d877e080-b2c0-11e8-8054-0966cfa3213b.png)